experimental/generation/generator/templates/standard/generator.lg

```diff
@@ -0,0 +1,359 @@
+> This file contains templates to help with the generation process
```

Comment:
Probably move this file to the parent directory and change all the references to point to it and add what you need for qna.  It looks like the one in professional is pretty much the same as this one.